### ISSUE TITLE
personality LTD squirrel ID error

### ISSUE THREAD

**Issue by martinig** [KluaneRedSquirrelProject/KRSP-Database-Errors#109]
I am helping one of Ben's students enter data in the personality LTD.  After speaking with Ben, he advised me to post an issue for this squirrel here and see if I can get help finding out where the typos might be.

There is a squirrel that she has with this info that does not seem to exist:

squirrel id: 100071 (clearly not possible)
sex: F
assay date: May 18, 2023
assay time: 08h32
grid: KL
left tag: M8270
right tag: M8271

@agmcadam Is there a way to tag Bruna in this so she can check books? If no, I can email her. I just can't seem to find her in the @mentions on here.

**Comment by agmcadam** [comment]
Have you searched for those tag numbers in 2023?
Andrew

On Nov 15, 2023, at 9:02 PM, April Martinig ***@***.***> wrote:



I am helping one of Ben's students enter data in the personality LTD.

There is a squirrel that she has with this info that does not seem to exist:

squirrel id: 100071 (clearly not possible)
sex: F
assay date: May 18, 2023
assay time: 08h32
grid: KL
left tag: M8270
right tag: M8271

@agmcadam<https://github.com/agmcadam> Is there a way to tag Bruna in this so she can check books? If no, I can email her. I just can't seem to find her in the @mentions on here.

**Comment by martinig** [comment]
Yes, I've also tried the tag numbers. I've checked the trapping LTD, litters LTD, and juvenile LTD.

**Comment by agmcadam** [comment]
Did you look in the 2023 database?

On Nov 15, 2023, at 9:10 PM, April Martinig ***@***.***> wrote:

﻿

Yes, I've also tried the tag numbers. I've checked the trapping LTD, litters LTD, and juvenile LTD.

**Comment by martinig** [comment]
I'm sorry, but I don't think I understand what database the 2023 database is. Am I able to get to this through the KRSP library in R?

**Comment by martinig** [comment]
@agmcadam Just following up about above. I'd love to look in the database you mentioned. :)

**Comment by agmcadam** [comment]
That squirrel_id is correct for those tag numbers.
